### ISSUE TITLE
Why is the recall rate on SYSU and LEVIR-CD+ slightly lower than the results in the paper?

### ISSUE THREAD

**Issue by WeiGuaa** [ChenHongruixuan/ChangeMamba#119]
Below are my test results on MambaBCD_Base on various datasets：

- LEVIR-CD+
`Racall rate is 0.8683298747623326, Precision rate is 0.9027817608660338, OA is 0.9908254913900091, F1 score is 0.8852207366052336, IoU is 0.7940771466357627, Kappa coefficient is 0.8804443997732503`

- SYSU
`Racall rate is 0.7899904655247997, Precision rate is 0.8812092990710301, OA is 0.9253595733642578, F1 score is 0.8331103906854184, IoU is 0.7139581876770489, Kappa coefficient is 0.7852250435534568`

- WHU-CD
`Racall rate is 0.923185768461217, Precision rate is 0.9629132885625522, OA is 0.9960064266038977, F1 score is 0.9426311316499795, IoU is 0.8914875024842708, Kappa coefficient is 0.9405632892518304`

The above test results show that except for the recall rate on SYSU and LEVIR-CD+ which is lower than the results in the paper, the results of other indicators are better than the paper. I am a little confused about why the test results with slightly lower recall rate appear?

Below are my test scripts on various data sets:
- LEVIR-CD+
`python script/infer_MambaBCD.py  --dataset 'LEVIR-CD+' \
                                 --model_type 'MambaBCD_Base' \
                                 --test_dataset_path '~/sdb/LEVIR-CD+/levir-cd+/test/' \
                                 --test_data_list_path '~/MambaCD/changedetection/datasets/data_name_list/LEVIR-CD+/test_list.txt' \
                                 --cfg '~/MambaCD/changedetection/configs/vssm1/vssm_base_224.yaml' \
                                 --resume '~/MambaCD/pretrained_weight/MambaBCD_Base_LEVIRCD+_F1_0.8823.pth'`

- SYSU
`python script/infer_MambaBCD.py  --dataset 'SYSU' \
                                 --model_type 'MambaBCD_Base' \
                                 --test_dataset_path '~/sdb/wjx/SYSU-CD/sysu-cd/test/' \
                                 --test_data_list_path '~/MambaCD/changedetection/datasets/data_name_list/SYSU/test_list.txt' \
                                 --cfg '~/MambaCD/changedetection/configs/vssm1/vssm_base_224.yaml' \
                                 --resume '~/MambaCD/pretrained_weight/MambaBCD_Base_SYSU_F1_0.8331.pth'`

- WHU-CD
`python script/infer_MambaBCD.py  --dataset 'WHU-CD' \
                                 --model_type 'MambaBCD_Base_7_25_16' \
                                 --test_dataset_path '~/sdb/wjx/WHU-CD/whu-cd/test/' \
                                 --test_data_list_path '~/MambaCD/changedetection/datasets/data_name_list/WHU-CD/test_list.txt' \
                                 --cfg '~/MambaCD/changedetection/configs/vssm1/vssm_base_224.yaml' \
                                 --resume '~/MambaCD/pretrained_weight/MambaBCD_Base_WHU_F1_0.9419.pth'`

Finally, I would like to ask a question. The pre-trained model weights （MambaBCD_Base.pth） downloaded from [Zenodo](https://zenodo.org/records/14037769) for each dataset seem to be different from the model weights used in the test experiment in the paper? Because I found that the result mask of BCD is slightly different from the example mask in the paper.

I hope to get your answer. Thank you very much for your outstanding work!

**Comment by ChenHongruixuan** [comment]
Hi @WeiGuaa ,

Thanks a lot for your interest to ChangeMamba. 

As we indicated in our README file: 
> Please note that the **code we uploaded was reorganised and collated**. The models below were also trained using the reorganised code and therefore accuracy may not perfectly match the original paper. In most cases, the accuracy is **higher** than that in the paper.

Best,

**Comment by WeiGuaa** [comment]
> Hi [@WeiGuaa](https://github.com/WeiGuaa) ,
> 
> Thanks a lot for your interest to ChangeMamba.
> 
> As we indicated in our README file:
> 
> > Please note that the **code we uploaded was reorganised and collated**. The models below were also trained using the reorganised code and therefore accuracy may not perfectly match the original paper. In most cases, the accuracy is **higher** than that in the paper.
> 
> If you find ChangeMamba is valuable to your research, please kindly consider give this repo a star :)
> 
> Best,

Thanks for your reply. I have checked the notes in README. Regarding the test results (other metrics performed well, but the recall rate was slightly reduced), I wonder if this is in line with the expected results?

**Comment by WeiGuaa** [comment]
> Hi [@WeiGuaa](https://github.com/WeiGuaa) ,
> 
> That is quite normal. Generally we care about F1 score and Kappa coefficient, right? But if you are concerned about this metric, maybe you can apply some loss functions ro make some improvements on our model.
> 
> Best,

Understood, thank you for your response!

**Comment by ChenHongruixuan** [comment]
great! if changemamba is useful to your research, please kindly considering give this repo a ⭐
